coding-intro/coding-intro-core/core-concepts/do-it-yourself.md

```diff
@@ -0,0 +1,39 @@
+---
+author: abdullah
+tags:
+  - coding
+type: normal
+category: coding
```

Comment:
you also need to change the topic's language to python ([here](https://github.com/enkidevs/curriculum/blob/master/coding-intro/README.md))

Reply:
Got it, will do that locally and push it.